### DISCUSSION TITLE
Welcome to haxo-rs Discussions!

### DISCUSSION THREAD

**Discussion by jcard0na** [cardonabits/haxo-rs#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by raychmond** [comment]
Hi @jcard0na 

I notice you built the control commands to change the bank. How do I do to have a pressure < -10? 

Thanks!

**Reply by jcard0na** [reply]
Oh, just draw air instead of blowing.  
So to enter a command:

1. Suck air in while holding low Bb fingering, then
2. Suck air in again with the fingerings below to execute a command:
low C: shut down haxophone
low F: change sound to previous instrument
low G: change sound to next instrument

After each command you will hear a loud confirmation sound.

**Reply by raychmond** [reply]
Thank you Javier. 

I finally figured out that I actually were using the service to run the program, not "sudo /usr/local/bin/haxo001". If I disable the haxo.service, and start the program using "sudo /usr/local/bin/haxo001", it won't have sound. I will look into this later because this is not the topic here.
Given I'm using the service, where the SoundFont is TenorSax2.sf2 and bank is 0, when I change the bank using the command , it won't work because that SoundFont may only have one bank. 
So I change the sf2 to "FluidR3_GM.sf2" and bank number to 67 in the service, it works perfectly. I have tried many different banks successfully. 
I will add my software experience in another thread soon.
Thank you!

**Reply by jcard0na** [reply]
That's good to know.  I believe TenorSax2 has only sax sounds that sound very similar.  That might trick you into thinking that the "instrument" did not change when it actually changed just slightly.  Could that be what happened here?

By the way, where did you get `TenorSax2.sf2` from?  Contrary to [FluidR3_GM](https://member.keymusician.com/Member/FluidR3_GM/index.html), I could not find a clear license on that file, that's why I did not check it in.  But I found it had better sax sound than `FluidR3_GM.sf2`

**Reply by hortense667** [reply]
Thank you as well. Merry Christmas!

**Comment by raychmond** [comment]
You are right. I barely recognize the difference in TensorSax2. I downloaded [here](https://musical-artifacts.com/artifacts?formats=sf2&tags=tenor+saxophone). It says open source on a donation basis. I'm not quite sure if it's legally licensed to use or not to be honest. Yes, it has better sax sound than FluidR3_GM.sf2
